src/main/java/com/iexec/worker/result/ResultService.java

```diff
@@ -181,6 +181,7 @@ public String uploadResultAndGetLink(WorkerpoolAuthorization workerpoolAuthoriza
             return "";
         }
 
+        final String resultProxyURL = task.getResultStorageProxy();
```

Comment:
As this is only used once, put this declaration next to its use. This will help the JVM to clean the variable when no more useful.